src/diffusers/pipelines/score_sde_ve/pipeline_score_sde_ve.py

```diff
@@ -40,7 +41,7 @@ def __call__(self, batch_size=1, num_inference_steps=2000, generator=None, torch
 
             sample, sample_mean = output["prev_sample"], output["prev_sample_mean"]
 
-        sample = sample.clamp(0, 1)
+        sample = sample_mean.clamp(0, 1)
```

Comment:
cc @natolambert I think previously we returned the mean here

Reply:
We did, I wasn't sure why. Want me to double check the original implementation?